pageserver/src/walreceiver/connection_manager.rs

```diff
@@ -586,7 +591,7 @@ impl WalreceiverState {
                     if latest_interaciton > self.wal_connect_timeout {
                         return Some(NewWalConnectionCandidate {
                             safekeeper_id: new_sk_id,
-                            wal_source_connstr: new_wal_source_connstr,
+                            wal_source_connconf: new_wal_source_connstr,
```

Comment:
A bit inconsistent naming, variable remains to be named `new_wal_source_connstr`